backup/global_variables.go

```diff
@@ -27,7 +28,7 @@ const (
 	Deferred
 	Complete
 	PG_LOCK_NOT_AVAILABLE = "55P03"
-	ENUM_TYPE_OID = 3500
+	ENUM_TYPE_OID         = 3500
```

Comment:
it's not relevant to this commit

Reply:
This was done by the default formatter in the VScode when saving the file.